feat: add generator skip option for table columns

galexrt opened the pull request:
This is a quick implementation of having a skip option for table columns/fields in the generator for the feature request I created #435.
This might not be optimal but the easiest way I found to implement this feature.

Resolves #435

go-jet commented:
Hi @galexrt, 
Yeah, it makes sense to have such a feature.
Not sure if it is intentional, with this change you're skipping only SQL builder columns, not model fields.
Could you also add a test here: https://github.com/go-jet/jet/blob/f9ed359785fcff71fbf1e48dfa3b8f5862d98302/tests/postgres/generator_template_test.go#L451

galexrt commented:
I have added a test and due to it being easier to handle skipped fields for the `column-list`, I had added a template function `columnList` to render the list via `strings.Join`.

go-jet commented:
Thanks, looks good. :+1:
